GitExtensions/Project.Build.targets

```diff
@@ -8,12 +8,12 @@
     ============================================================
     -->
   <Target Name="_CopyTranslations" AfterTargets="AfterBuild">
-    <_GetAvailableTranslationsCodeTask
+    <!-- <_GetAvailableTranslationsCodeTask
```

Comment:
💡 Can you link commented-out code to specific issues in cases where behavior changed?

Reply:
This is part of outstanding "Packaging" task (in the 1st post)